### PR TITLE
adding DL nodes (split from FF nodes)

### PR DESCRIPTION
**Pull request by vishalchangrani**
The nodes run by the Flow team are being split between Flow Foundation and DapperLabs.
This transaction adds the DapperLabs nodes to the approved list of nodes. The Flow Foundation nodes are being unstaked separately.

Master sheet: [DLNodes google sheet](https://docs.google.com/spreadsheets/d/1ZK5RDD9wZkSlQkhh8m3h7ywYd1OR0jpoFFdOJUTdFAM/edit?usp=sharing)

### CODE REVIEW

File: transactions/set-approved-id-list/2023/nov-6/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Add DapperLab node IDs
+
+> Nov 6th, 2023 
+
+The current nodes run by the Flow team is being split between FF and DL. This transaction is to add the DL nodes to the approved list of nodes.
+
+92 nodes in total
```
[the remark this answers is not in the document]
Comment (KshitijChaudhary666) [reply transactions/set-approved-id-list/2023/nov-6/README.md:7]: Collection | 48
Consensus | 17
Execution | 2
Verification | 25
--- Sum | 92

**Review by KshitijChaudhary666 — APPROVED** [review]

**Comment by joshuahannan** [comment]
We'll need to update the arguments to not include all the VNs that we weren't able to stake today

File: transactions/set-approved-id-list/2023/nov-6/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
...%20%20%20%20%20%20%22value%22:%20%22c15523db34e1e054e7fdf4d393ac5044d4802a256a0dc1a929625dd3dae15617%22%20%20%20%20%20%20%20%20%20%20%20%20%20%7D,%20%20%20%20%20%20%20%20%20%20%20%20%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22type%22:%20%22String%22,%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22value%22:%20%223327847b85e36fdfabc320eaa3019685fbb69baf077fade7318b7b4be7e91dce%22%20%20%20%20%20%20%20%20%20%20%20%20%20%7D,%20%20%20%20%20%20%20%20%20%20%20%20%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22type%22:%20%22String%22,%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22value%22:%20%224a9db4104cdf282f1a9d440c4f33324f280d3d33395ef95d45309baddfb14312%22%20%20%20%20%20%20%20%20%20%20%20%20%20%7D,%20%20%20%20%20%20%20%20%20%20%20%20%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22type%22:%20%22String%22,%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22value%22:%20%227bc70456cd33d82f7cd127f6684aa52d61f33219d612bad96046db923e7a27db%22%20%20%20%20%20%20%20%20%20%20%20%20%20%7D%20%20%20%20%20%20%20%20%20%5D%20%20%20%20%20%7D%20%5D&acct=0x8624b52f9ddcd04a&limit=9999) for the `add_approved_id_list.cdc` transaction with the given args.
```
[the remark this answers is not in the document]
Comment (vishalchangrani) [reply transactions/set-approved-id-list/2023/nov-6/README.md:107]: Thanks Josh

**Comment by vishalchangrani** [comment]
> We'll need to update the arguments to not include all the VNs that we weren't able to stake today

yup, removed the last 6 VNs that were not staked.

File: transactions/set-approved-id-list/2023/nov-6/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vishalchangrani) [reply transactions/set-approved-id-list/2023/nov-6/README.md]: Collection | 48
Consensus | 17
Execution | 2
Verification | ~~25~~ 19
--- Sum | ~~92~~

revised numbers

**Review by joshuahannan — APPROVED** [review]
I doubled checked to make sure all the node IDs match and it looks good

**Comment by vishalchangrani** [comment]
> I doubled checked to make sure all the node IDs match and it looks good
Thankyou!

**Pull request merged**